ros/angel_system_nodes/angel_system_nodes/task_monitor.py

```diff
@@ -65,7 +66,7 @@ class CoffeeDemoTask():
     Steps are listed under v1.2 here:
     https://docs.google.com/document/d/1MfbZdRS6tOGzqNSN-_22Xwmq-huk5_WdKeDSEulFOL0/edit#heading=h.l369fku95vnn
     """
-    def __init__(self):
+    def __init__(self, task_steps_file):
```

Comment:
Please add a little documentation on what the format of this file is expected to be.